chat-client/src/client/mynahUi.ts

```diff
@@ -131,6 +143,60 @@ export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [Myna
             }
             messager.onVote(payload)
         },
+        onSendFeedback: (tabId, feedbackPayload, eventId) => {
+            if (feedbackPayload.comment !== undefined) {
```

Comment:
I think we don't need this if check and comment can be empty 

On mynahui feedback form, you choose an issue type from the dropdown and optionally type some comments https://github.com/aws/mynah-ui/blob/main/docs/PROPERTIES.md#onsendfeedback

Reply:
ah, I missed that detail, will update the logic